Makefile

```diff
@@ -28,7 +29,7 @@ rhel_iso: iso_creation
 
 .PHONY: iso_creation
 iso_creation: init
-	handle_user_data='$(HANDLE_USER_DATA)' cert_gen='$(CERT_GEN)' version='$(VERSION)' envsubst < $(KICKSTART_TEMPLATE) > $(BUILD_DIR)/$(KICKSTART_FILE)
+	handle_user_data='$(HANDLE_USER_DATA)' cert_gen='$(CERT_GEN)' version='$(VERSION).$(TODAY)' envsubst < $(KICKSTART_TEMPLATE) > $(BUILD_DIR)/$(KICKSTART_FILE)
```

Comment:
We should use BUILD_ID https://www.freedesktop.org/software/systemd/man/os-release.html#BUILD_ID=